backend/lib/models.ml

```diff
@@ -8,13 +8,25 @@ module Benchmark = struct
     commit : string;
     branch : string option;
     pull_number : int option;
-    name : string option;
-    data : Yojson.Safe.t;
+    benchmark_name : string option;
+    test_name : string;
+    metrics : Yojson.Safe.t;
   }
 
   let make ~run_at ~duration ~repo_id ~commit ?branch ?pull_number data =
-    let name = Yojson.Safe.Util.(member "name" data |> to_string_option) in
-    { run_at; duration; repo_id; commit; branch; pull_number; name; data }
+    let test_name = Yojson.Safe.Util.(member "name" data |> to_string) in
+    let metrics = Yojson.Safe.Util.(member "metrics" data) in
+    {
+      run_at;
+      duration;
+      repo_id;
+      commit;
+      branch;
+      pull_number;
+      benchmark_name = None;
```

Comment:
I don't know where we should grab this data from? I didn't see any `benchmark_name` field in the benchmark results..

Reply:
This can be obtained from the optional top-level `name` field. Tried to clarify this in the docs: https://github.com/ocurrent/current-bench/commit/805c8bc0bad18ea704d4bd14a567bfa080644639.